app/controllers/api/v1x0/templates_controller.rb

```diff
@@ -1,24 +1,15 @@
 module Api
   module V1x0
     class TemplatesController < ApplicationController
-      include Api::V1x0::Mixins::IndexMixin
-      include Api::V1x0::Mixins::RBACMixin
-
-      before_action :read_access_check, :only => %i[show]
+      include Mixins::IndexMixin
 
       def show
-        template = Template.find(params.require(:id))
+        template = policy_scope(Template).find(params.require(:id))
```

Comment:
why can't we find the template and `authorize` like in other controller?

Reply:
Yes, we can, but hey are the same for `template`